doc/faq.rst

```diff
@@ -0,0 +1,90 @@
+.. _faq:
+
+.. |br| raw:: html
+
+   <div style="line-height: 0; padding: 0; margin: 0"></div>
+
+
+Troubleshooting and FAQ
+=======================
+
+.. rubric:: :fa:`quora`. I want to train a deep learning model. Do you support this?
+
+.. dropdown:: :fa:`comments`  Answer
+    :container: + shadow
+
+    ``rsmtool`` supports a large number of :ref:`learners<model>` but not deep learning algorithms. However, you can still use ``
+    rsmeva`` :ref:`rsmeval<usage_rsmeval>` to obtain a comprehensive evaluation report for the 
+    from any model!
+
+.. rubric:: :fa:`quora`. How do I make changes to the report?
+
+.. dropdown:: :fa:`comments`  Answer
+    :container: + shadow
+
+    The reports generated by ``rsmtool``  and ``rsmeval`` are designed to be fully customizable. 
+    You can:
+
+    - Omit sections using :ref:`general_sections`<general_sections>` field;
+
+    - Change the order of the sections using :ref:`section_order<section_order>` field; 
+
+    - Create your own :ref:`"custom" sections<custom_notebooks>`.
+
+
+.. rubric:: :fa:`quora`. I got the following error:
+ ``No responses remaining after filtering out non-numeric feature values. No further analysis can be run.`` What happened?
+
+.. dropdown:: :fa:`comments`  Answer
+    :container: + shadow
+
+	``rsmtool`` is designed to work with numeric features only. Non-numeric values including missing 
+	values are filtered out. Some of the common reasons for the above error are:
+
+	- The human score column or one of the features only contains non-numeric values. You can either exclude this feature or convert it to one-hot encoding. 
+
+	- You have features with sparse representation. The solution is to replace missing values with zeros. Note that this applies even if you use ``.jsonlines`` format. If you are using ``rsmtool`` with sparse format, let us know if `this issue <https://github.com/EducationalTestingService/rsmtool/issues/480>`_ will make your life easier. 
+
+	- You have a lot of missing feature values and none of the responses has numeric features for every single feature. Inspect :ref:`*_excluded_responses<rsmtool_excluded_responses>` to see what responses have been excluded. 
+
+.. rubric:: :fa:`quora`. Can you pass a set of learners to the model key in the config file or do you do a separate run for each leaner you want to look at? 
+
+.. dropdown:: :fa:`comments`  Answer
+    :container: + shadow
+
+	``rsmtool`` can't do multiple learners yet. If you want to do multiple learners, you can use RSMTool API instead of the command line.
```

Comment:
```suggestion
	``rsmtool`` can't work with multiple learners yet. If you want to use multiple learners, you should use the RSMTool API instead of the command line.
```